jujugui/static/gui/src/app/views/utils.js

```diff
@@ -1577,15 +1577,18 @@ YUI.add('juju-view-utils', function(Y) {
       return;
     }
 
-    jem.newModel(
-      users.jem.user,
+    controllerAPI.createModel(
       model,
-      credential,
+      // Check that this is a user tag, if not turn it into one.
```

Comment:
I'd like for this logic to be elsewhere, and for functions to clearly define what they receive. A user name and a user tag are two completely different kind of values, and we should be clear here about what's expected, and callers are responsible of providing the right value (either a tag or a name), otherwise we'll end up putting this logic everywhere.